airflow_blogpost.txt

```diff
@@ -0,0 +1,110 @@
+Today, we're announcing an integration between Versatile Data Kit and Apache Airflow, a popular workload scheduler. This blog post serves to provide an overview of the relevant parts of these two projects, how they integrate, and how to set up an instance of Airflow which will trigger deployed VDK jobs.
+
+Apache Airflow is "a platform to programmatically author, schedule and monitor workflows". It allows users to schedule workloads to be ran periodically, and also to build more complex workload structures by expressing them as directed acyclic graphs. The nodes of these graphs represent individual workloads, while the edges represent dependencies between workloads, meaning that a workload which is `pointed at` by another workload will only trigger if the previous workload completes successfully.
```

Comment:
Everything is one long section. Split it into multiple sections/parts each with its own title.